scripts/quality_test/quality_test.py

```diff
@@ -508,37 +271,42 @@ def _extract_missing_pii_annotations(filename):
     return (uncovered_models, pii_check_passed, full_log)
 
 
-@task
-@needs('pavelib.prereqs.install_python_prereqs')
-@cmdopts([
-    ("report-dir=", "r", "Directory in which to put PII reports"),
-])
-@timed
-def run_pii_check(options):
+def run_pii_check():
     """
     Guarantee that all Django models are PII-annotated.
     """
+    REPO_ROOT = repo_root()
+    REPORT_DIR = REPO_ROOT / 'reports'
     pii_report_name = 'pii'
-    default_report_dir = (Env.REPORT_DIR / pii_report_name)
-    report_dir = getattr(options, 'report_dir', default_report_dir)
+    default_report_dir = (REPORT_DIR / pii_report_name)
+    report_dir = default_report_dir
     output_file = os.path.join(report_dir, 'pii_check_{}.report')
     env_report = []
     pii_check_passed = True
     for env_name, env_settings_file in (("CMS", "cms.envs.test"), ("LMS", "lms.envs.test")):
         try:
-            print()
             print(f"Running {env_name} PII Annotation check and report")
             print("-" * 45)
             run_output_file = str(output_file).format(env_name.lower())
-            sh(
-                "mkdir -p {} && "  # lint-amnesty, pylint: disable=duplicate-string-formatting-argument
-                "export DJANGO_SETTINGS_MODULE={}; "
-                "code_annotations django_find_annotations "
-                "--config_file .pii_annotations.yml --report_path {} --app_name {} "
-                "--lint --report --coverage | tee {}".format(
-                    report_dir, env_settings_file, report_dir, env_name.lower(), run_output_file
-                )
+            os.makedirs(report_dir, exist_ok=True)
+            command = (
+                f"export DJANGO_SETTINGS_MODULE={env_settings_file};"
+                f"code_annotations django_find_annotations"
+                f"--config_file .pii_annotations.yml --report_path {report_dir} --app_name {env_name.lower()}"
+                f"--lint --report --coverage | tee {run_output_file}"
+            )
+            result = subprocess.run(
+                command,
+                shell=True,
+                check=False,
```

Comment:
On all of these, we need `check=True` so that a Python exception is raised when the underlying command fails. Otherwise, if there is an error in the command, the check will silently pass.